### PR TITLE
[Delivers #93656370] Integrates @24thcentury's home page

### PR DESCRIPTION
**Pull request by cmc333333**
Swapped out a few of the images and text:
screen shot 2015-05-07 at 11 51 57 am

Also integrated the main nav. Not perfectly responsive, but better than nothing.
header

### CODE REVIEW

**Comment by cmc333333** [comment]
Alerts:
screen shot 2015-05-07 at 12 20 48 pm

**Comment by phirefly** [comment]
Regarding alerts, I'd be inclined to put it at the top to address the ugly banding that happens, but I know we have had some banner blindness syndrome. So :+1: as first pass.

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,35 +1,96 @@
...
+					<ul class="actions uniform">
+						<li><a href="https://github.com/18F/C2" class="button special">Fork us on GitHub</a></li>
+					</ul>
+				</div>
+				</div>
+				</div>
+			</section>
+
+	</body>
+</html>
```
Comment (afeld) [inline app/views/home/index.html.erb]: Needs indentation cleanup.

**Comment by cmc333333** [comment]
Indentation a problem? Haml it is.

**Comment by phirefly** [comment]
:shipit:

**Pull request merged**